modules/google-speech/src/backend/typings.ts

```diff
@@ -0,0 +1,25 @@
+import { protos as speechProtos } from '@google-cloud/speech'
+import { protos as textToSpeechProtos } from '@google-cloud/text-to-speech'
+import { GoogleSpeechClient } from './client'
+
+export interface Clients {
+  [botId: string]: GoogleSpeechClient
+}
+
+export type EnumDictionary<T extends string | symbol | number, U> = {
+  [K in T]: U
+}
+
+export type IRecognitionConfig = speechProtos.google.cloud.speech.v1.IRecognitionConfig
+export type IRecognitionAudio = speechProtos.google.cloud.speech.v1.IRecognitionAudio
+export const AudioEncoding = speechProtos.google.cloud.speech.v1.RecognitionConfig.AudioEncoding
+
+export type ISynthesizeSpeechRequest = textToSpeechProtos.google.cloud.texttospeech.v1.ISynthesizeSpeechRequest
```

Comment:
When using types, please avoid adding the "I" prefix before... We know it's an interface, prefixing with the type is an old java paradigm

Reply:
I agree with you I simply didn't want to rename the names of the interfaces!